README.md

```diff
@@ -3,11 +3,33 @@
 [![Crates.io](https://img.shields.io/crates/v/resend-rs)](https://crates.io/crates/resend-rs)
 [![docs.rs](https://img.shields.io/docsrs/resend-rs)](https://docs.rs/resend-rs)
 
-A *very* minimal [Resend](https://resend.com) client for sending emails.
+A minimal [Resend](https://resend.com) client for sending emails.
 
-Emails are sent via the [`resend_client::ResendClient`] which provides both a synchronous and
-asynchronous send method. The two are mutually exclusive and accessible via the `async` feature. The
-crate uses [`reqwest`](https://github.com/seanmonstar/reqwest) internally.
+Emails are sent via the [`Resend`] client which provides both a
```

Comment:
Just a small note here, I had "[`resend_client::ResendClient`]" before because I use the readme in the [docs](https://docs.rs/resend-rs/latest/resend_rs/) and it links to the struct properly. More noting this down for myself if anything, it is probably better we move/copy the code examples in the docs directly instead of the readme.

Reply:
Yeah, I've reexported the `Client` into the lib so that docs should look fine now. 
Agree, I'll move code examples into the `lib.rs`.

The main issue right now is that some of the types generated with `openapi-generator -i` [resend.yaml][yaml] appear to be either outdated (according to the [API reference][API]) (e.g., available region list) or not entirely correct (e.g. options on required params).

[yaml]: https://raw.githubusercontent.com/resend/resend-openapi/main/resend.yaml
[API]: https://resend.com/docs/api-reference